src/App.js

```diff
@@ -1,22 +1,47 @@
 import React from "react";
-import "./App.css";
-import ToggleSwitch from "./components/ToggleSwitch";
+import styled from "styled-components";
+import SettingsBox from "./components/SettingsBox";
+import Head from "./components/Head";
+
+// https://coolors.co/282c34-4fbe79-39a2ae-434f4f-7a9e9f
+
+const Playground = styled.div`
+  height: 100vh;
+  width: 100vw;
+  background-color: #282c34;
+`;
+
+const TheGrid = styled.div`
+  height: fit-content;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+  grid-template-rows: repeat(auto-fit, minmax(120px, 1fr));
+`;
 
 function App() {
   const [settings, setSettings] = React.useState({
-    darkmode: false
+    darkmode: false,
```

Comment:
I would create a variable like `defaultSettings` to  make this component smaller. You can import this variable from a different file.